src/components/element-editor/conditional-rendering/conditional-rendering.tsx

```diff
@@ -149,7 +149,7 @@ export const ConditionalRendering = () => {
             title="Conditional Rendering"
             buttonLabel="condition"
             disabledButton={
-                getElementOptions().length === 0 || invalidAndEmptyFields
+                Object.values(elements).length === 1 || invalidAndEmptyFields
```

Comment:
this prevents the button from being disabled even if there are invalid fields.

shouldnt it be disabled if there are no other elements?